cmake/thirdparty/get_rapids-triton.cmake

```diff
@@ -40,7 +40,19 @@ endfunction()
 # Change pinned tag here to test a commit in CI
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
+set (RAPIDS_FORK https://github.com/rapidsai/rapids-triton.git)
+set (REPO_TAG branch-${RAPIDS_DEPENDENCIES_VERSION})
+# if Triton tag and organization is specified, change the fork and the repo
+if (NOT TRITON_REPO_ORGANIZATION STREQUAL "https://github.com/triton-inference-server")
+  set (RAPIDS_FORK ${TRITON_REPO_ORGANIZATION}/rapids_triton.git)
```

Comment:
My bad for not catching the typo. I will submit a fix right now.

Reply:
Hi @hcho3 it's not a typo the mirrored (pb) version is on rapids_triton. We could also change the mirrored gitlab name. @mc-nv @nv-kmcgill53  thoughts?

Reply:
In that case, users should be able to specify the full URL, not just the org prefix.